src/dmd/aggregate.h

```diff
@@ -277,7 +270,7 @@ class ClassDeclaration : public AggregateDeclaration
     int cppDtorVtblIndex;               // slot reserved for the virtual destructor [extern(C++)]
     bool inuse;                         // to prevent recursive attempts
 
-    Abstract isabstract;                // 0: fwdref, 1: is abstract class, 2: not abstract
+    ThreeState isabstract;              // 0: none, 1: not abstract, 2: is abstract class
```

Comment:
Repeating the integer values of the enum members forms a typical dead code comment that easily goes out of sync with the actual code. Seems like a relic from before proper D enums were used.

Reply:
ok